### ISSUE TITLE
Add support for removing appcmd settings

### ISSUE THREAD

**Issue by spuder** [chef-cookbooks/iis#207]
Requesting the ability to remove a config using `appcmd clear config ...`

Intended use case, add and remove machine keys

Powershell command to add a machine key
```powershell
# Add machine key
appcmd.exe set config /commit:WEBROOT /section:machineKey /validation:AES /validationKey:1234 /decryptionKey:5678
```

Chef lwrp to add machine key
```chef
iis_config "/commit:WEBROOT /section:machineKey /validation:#{node['foo']['static_machine_key']['validation']} /validationKey:#{node['foo']['static_machine_key']['validation_key']} /decryptionKey:#{node['foo']['static_machine_key']['decryption_key']}"
  action :config
end
```

Powershell command to remove machineKey config from machine.config
```powershell
# Remove machine key
appcmd clear config /commit:WEBROOT /section:machineKey
```

Chef lwrp to remove machinekey from machine.config
```chef
iis_config "/commit:WEBROOT /section:machineKey"
  action :clear
end
```

I'll admit I've never worked with LWRP's so I'm in a bit over my head. Feedback appreciated. 

Down the road, I think `:config` should be aliased with `:set` for consistency.

**Comment by EasyAsABC123** [comment]
@spuder I updated my comment, so sorry on the mis-understanding.  I think the simple cleanup should be perfect...Then do a `git rebase -i X` and squash the 3 commits down to 1.

**Comment by EasyAsABC123** [comment]
merged #208
